plugins/i18n_navigation/frontend.class.php

```diff
@@ -195,18 +195,20 @@ private static function getMenuImpl($breadcrumbs, $currenturl, $url, $levels, $s
           global $filters;
           $params = array($childurl, $pages[$childurl]['parent'], 
                           preg_split('/\s*,\s*/', html_entity_decode(stripslashes(trim(@$pages[$childurl]['tags'])), ENT_QUOTES, 'UTF-8')));
-          foreach ($filters as $filter)  {
-            if ($filter['filter'] == I18N_FILTER_VETO_NAV_ITEM) {
-              if (call_user_func_array($filter['function'], $params)) {
-                $showIt = false; 
-                break;
+          if(!empty($filters)){
+            foreach ($filters as $filter)  {
+              if ($filter['filter'] == I18N_FILTER_VETO_NAV_ITEM) {
+                if (call_user_func_array($filter['function'], $params)) {
+                  $showIt = false; 
+                  break;
+                }
               }
             }
           }
         }
         if ($showIt) {
           $showChildren = !($show & I18N_FILTER_CURRENT) || in_array($childurl,$breadcrumbs);
-          $children = $showChildren ? self::getMenuImpl($breadcrumbs, $currenturl, $childurl, $levels-1, $show) : null;
+          $children = $showChildren ? self::getMenuImpl($breadcrumbs, $currenturl, $childurl, $levels-1, $show) : [];
```

Comment:
I suggest using `array()` instead of `[]` for compatibility with PHP < 5.4

Reply:
I don't have a problem with that, but do you think this has a meaning when the whole PHP 5.x line and even 7.0 versions are out of support?

Reply:
I don't care either, but as long as GS claims to be compatible with PHP 5.2 ...

Reply:
I agree with cnb also makes easier to read code